src/components/Navbar/index.tsx

```diff
@@ -154,6 +167,16 @@ export default class Navbar extends React.Component<any, any> {
       </MenuButton>
       )];
 
+    if (noTemplates && !dashboards) {
```

Comment:
If there is a problem with auth (perhaps I made a mistake with the settings) and I need to switch off auth I still would need access to the setup page to fix the settings. Should we update to the conditional so that its a request for the /setup page then proceed to render as normal. The setup page is FOC (flash of content) then is replaced by problem message.